### ISSUE TITLE
Get configured Header of Column

### ISSUE THREAD

**Issue by benno-mueller-saxess** [vaadin/vaadin-grid-flow#901]
In a Grid you can set a header for a column: `addColumn(...).setHeader("headerForThisColumn")`

We want to implement an export function that exports all data of the Grid to a file (CSV, Excel, ...). In order to make easy to use, we want to use the grid to get all column headers. So the user does not have to configure the column header twice (for the Grid and for the export), but only once (for the Grid).

Therefore we need to read the column headers of an existing and configured Grid. This is not possible in Vaadin 14 now.

**Comment by Legioth** [comment]
Duplicate of #567
